### PR TITLE
Create 2020-06-02-coreweekly-22-2020.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+* [#9](https://github.com/PrestaShop/traces/pull/9): Changed arguments for username & password in options, by [@Progi1984](https://github.com/Progi1984)
+
+
+### example_module_mailtheme
+* [#6](https://github.com/PrestaShop/example_module_mailtheme/pull/6): Bump jquery from 3.4.1 to 3.5.0. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#5](https://github.com/PrestaShop/example_module_mailtheme/pull/5): Bump acorn from 6.1.1 to 6.4.1. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#4](https://github.com/PrestaShop/example_module_mailtheme/pull/4): Bump lodash from 4.17.11 to 4.17.15. Thank you [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### ps_currencyselector
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: No "pretty" name

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+### Gamification module
+* [#73](https://github.com/PrestaShop/gamification/pull/73): Rename function name to avoid conflicts, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Traces
+* [#9](https://github.com/PrestaShop/traces/pull/9): Changed arguments for username & password in options, by [@Progi1984](https://github.com/Progi1984)
+
+
+### example_module_mailtheme
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md:188]: No "pretty" name

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+* [#39](https://github.com/PrestaShop/QANightlyResults/pull/39): Variability count now uses browser and campaign too, by [@SimonGrn](https://github.com/SimonGrn)
+
+
+### Email Alerts module
+* [#53](https://github.com/PrestaShop/ps_emailalerts/pull/53): Rebase master in dev for merging, by [@Progi1984](https://github.com/Progi1984)
+* [#52](https://github.com/PrestaShop/ps_emailalerts/pull/52): Update version to 2.2.0 & Some Project Improvements, by [@Progi1984](https://github.com/Progi1984)
+* [#43](https://github.com/PrestaShop/ps_emailalerts/pull/43): Improve account template. Thank you [@micka-fdz](https://github.com/micka-fdz)
+
+
+### gadwords
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: No "pretty" name

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+### Product Comments module
+* [#42](https://github.com/PrestaShop/productcomments/pull/42): Rebuild ratings loading on product list. Thank you [@Oksydan](https://github.com/Oksydan)
+
+
+### Email subscription module
+* [#35](https://github.com/PrestaShop/ps_emailsubscription/pull/35): Fixed the email when user is registering (to the site and the newsletter simultaneously). Thank you [@Codencode](https://github.com/Codencode)
+
+
+### vatnumber
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: No "pretty" name

Did you read the PR content before opening it ? 🤔

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+### ps_currencyselector
+* [#17](https://github.com/PrestaShop/ps_currencyselector/pull/17): Add condition to avoid duplicate currency name, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+### Contact informations module
+* [#32](https://github.com/PrestaShop/ps_contactinfo/pull/32): Fixed uninitialized string offset bug. Thank you [@joaolago1113](https://github.com/joaolago1113)
+
+
+### ps_buybuttonlite
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: No "pretty" name

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+
+### Email subscription module
+* [#35](https://github.com/PrestaShop/ps_emailsubscription/pull/35): Fixed the email when user is registering (to the site and the newsletter simultaneously). Thank you [@Codencode](https://github.com/Codencode)
+
+
+### vatnumber
+* [#11](https://github.com/PrestaShop/vatnumber/pull/11): [-] Increase curl timeout from 5 to 10 sec. Thank you [@vmsrg](https://github.com/vmsrg)
+
+
+### ps_categoryproducts
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: No "pretty" name

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]: what is a "pretty" name ?

Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### Currency selector
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md:188]: Nothing to do here, this is the title in the repos's readme.md

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### Google Ads
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### European VAT number
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 6 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### Buy button lite
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md (hunk 5 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### Buy button lite
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+
+### Email subscription module
+* [#35](https://github.com/PrestaShop/ps_emailsubscription/pull/35): Fixed the email when user is registering (to the site and the newsletter simultaneously). Thank you [@Codencode](https://github.com/Codencode)
+
+
+### European VAT number
+* [#11](https://github.com/PrestaShop/vatnumber/pull/11): [-] Increase curl timeout from 5 to 10 sec. Thank you [@vmsrg](https://github.com/vmsrg)
+
+
+### Buy button lite
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
### Products in the same category
```

File: news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md [hunk 8]
```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 22 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-06-02
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md]:
```suggestion
date:   2020-06-04 10:14:00
```

**Pull request merged**